lib/rspec/expectations/configuration.rb

```diff
@@ -56,6 +56,19 @@ def syntax=(values)
         end
       end
 
+      # Configures the maximum character length that RSpec will print while
+      # formatting an object
+      # @param [Fixnum] the length to format to
```

Comment:
@benoittgt this rdoc is wrong too :/ It needs to be `@param [Fixnum] length the number of characters to limit the formatted output to` (As the first word needs to match the variable).

Reply:
Today I learn this. Thanks